buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -703,6 +707,22 @@ class ClusterFormationTasks {
                     }
                 }
             }
+
+            nodes.forEach {node ->
+                if (node.nodeVersion.onOrAfter("7.0.0-SNAPSHOT")) {
+                    Collection<String> unicastHosts = new HashSet<>()
+                    nodes.forEach { otherNode ->
```

Comment:
I think this should only be the master-eligible nodes.

Reply:
I thought the same, but then I found this code https://github.com/elastic/elasticsearch/blob/master/buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterConfiguration.groovy#L71 => aren't all nodes master-eligible nodes in these tests?

Reply:
Ah now I read #27344, so I guess until we change that all nodes are master eligible indeed?

Reply:
You are correct: now that I look more closely it seems that all nodes in these tests are master-eligible. Don't mind me...